omod/src/main/java/org/openmrs/module/pihmalawi/RestUtils.java

```diff
@@ -38,4 +40,13 @@ public static ResponseEntity<Map<String, Object>> errorResponse(Throwable t) {
         data.put("errorMessages",errorMessages);
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(data);
     }
+
+    public static boolean isMacAddressAllowed(String macAddress) {
+        boolean allowed = false;
+        if (StringUtils.isNotBlank(macAddress)) {
+            String addresses = Context.getAdministrationService().getGlobalProperty(PihMalawiConstants.SYNC_ALLOW_MAC_ADDRESSES_GP_NAME);
+            allowed = StringUtils.isNotBlank(addresses) && addresses.toLowerCase().contains(macAddress.toLowerCase());
```

Comment:
I assume you want this to block all users by default?

Reply:
Yes, block all users by default. Is there more behind your question here?